README.md

```diff
@@ -87,9 +87,15 @@ export ODFUZZ_ASYNC_REQUESTS_NUM=10
 
 File path where the HTTPS certificate is stored if the service is requiring it.
 ```
-export ENV_ODFUZZ_CERTIFICATE_PATH=./cert.crt
+export ODFUZZ_CERTIFICATE_PATH=./cert.crt
 ```
 
+A default encoder and decoder for generated strings. The encoder is used for encoding strings which may contain special characters. The decoder is used then for decoding the string back to its original representation.
```

Comment:
should be better to use "generated url parts" instead of "generated strings"? the latter could have many possible meaning and even with this documentation I would not be sure what the configuration affects, if I would not spend such time in PR review.

similarly the env. variable should be ODFUZZ_URL_PART_ENCODER or something similar, that would point just by its name what it really affects.

Reply:
or correct if I am wrong.. I noticed that the encoder is not used for Edm.DateTime but the PR is marked as closes #40.
It is intentional?

It may be worth to write to documentation list of odata V2 types that are affected by this configuration option.

Reply:
In the past, ODfuzz had been encoding every single generated **part** of a **query** while building an URL. In this commit, ODfuzz encodes only a specific subset of generated **strings** coming straight from the generators. These strings shape one **query** when combined together.

The encoder is not used for Edm.DateTime because it is not necessary to use it there. What do we want to encode in Edm.DateTime? During generation, there should not be present any special characters because the OData protocol forbids that; on the contrary, Edm.String may contain multiple UTF-8 characters which must be encoded.

I can add a list of all affected Edm types to README.

Reply:
README update with list of affected types would be great & enough.